src/tests/unit/tests/injected/analyzers/tab-stops-analyzer.test.ts

```diff
@@ -249,6 +250,10 @@ describe('TabStopsAnalyzer', () => {
             testSubject.teardown();
             await flushSettledPromises();
             verifyAll();
+            tabStopRequirementActionMessageCreatorMock.verify(
+                m => m.automatedTabbingResultsCompleted(It.isAny()),
```

Comment:
Feels like we should be explicitly checking what the requirement results object looks like here.

Reply:
Good point, updated, thanks!